src/display/canvas.js

```diff
@@ -1275,6 +1281,21 @@ class CanvasGraphics {
     this.cachedCanvases.clear();
     this.cachedPatterns.clear();
 
+    for (const cache of this._cachedBitmaps) {
+      for (const result of cache.values()) {
+        if (result instanceof ImageBitmap) {
+          result.close();
```

Comment:
Are these `ImageBitmap`s the same ones that are cached in the API, in `PDFPageProxy.objs`?
Because if so, won't closing them here break the next rendering when e.g. zooming is done in the viewer for pages where [this code](https://github.com/mozilla/pdf.js/blob/fffce7910e63038cc15cd01522a768ee86e081be/src/display/api.js#L2800-L2803) *is not* being hit?

Naively, it seems to me that we should *not* be releasing `ImageBitmap` here and simply let the existing caching in the API handle that.